products-service/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "products-service",
+  "version": "1.0.0",
+  "description": "Serverless aws-nodejs-typescript template",
+  "main": "serverless.ts",
+  "scripts": {
+    "deploy": "sls deploy",
+    "test": "jest",
+    "swagger": "nodemon --config swagger.nodemon.json"
+  },
+  "engines": {
+    "node": ">=14.15.0"
```

Comment:
I'd restrict to a major node version only, unless there's a specific need to restrict to `>=14.15.0`

Reply:
I would prefer to leave it as is, because `package.json` was created by template. At least I don't see reason to investigate dependency tree to know why there is `>=14.15.0`.

Reply:
Yeah, I saw some other PRs and also encountered the same `>14.15.0` thing. Not critical of course